src/components/layout/Header.tsx

```diff
@@ -1,8 +1,8 @@
 import Image from 'next/image';
 import Link from 'next/link';
 import { useTheme } from 'next-themes';
-import logoDark from 'public/logo-dark.png';
-import logoLight from 'public/logo-light.png';
+import logoDark from '../../../public/logo-dark.png';
+import logoLight from '../../../public/logo-light.png';
```

Comment:
I wonder why this change is needed with the nextjs 14 update (I know it is needed though since this was one of the build errors. I tried fixing with an import like this, but I think they should be equivalent)
```suggestion
import logoDark from '@/../public/logo-dark.png';
import logoLight from '@/../public/logo-light.png';
```

Reply:
Oh yes, I should've clarified that I have no idea what I'm doing here.

My guess is that some default changed in this version of next (maybe some webpack module aliases thing?) and so the image loader doesn't work anymore with this kind of import 🤷‍♂️